exercises/set_intersection/python/main.py

```diff
@@ -0,0 +1,22 @@
+# Returns the intersection of two lists, tuples, or sets
+# Note True == 1 and False == 0
+def intersection(list1,list2):
+    try:
+        return set(list1) & set(list2)
+    except TypeError:
+        print("intersection: Inputs must be lists, tuples, or sets")
```

Comment:
usually in an error message printed it is useful to include the invalid input that caused the error so that a user of your functions can debug the issue easier